pkg/shared/remote/service-constants.json

```diff
@@ -9,7 +9,8 @@
       "quotaProductId": "RHOSAK",
       "trialQuotaProductId": "RHOSAKTrial",
       "longLivedQuotaProductId": "RHOSAKEval",
-      "resourceName": "rhosak"
+      "resourceName": "rhosak",
+      "enterpriseProductQuotaId": "RHOSAKCC"
```

Comment:
We'll also need to add this constant here https://github.com/redhat-developer/app-services-ui/blob/main/static/configs/service-constants.json